templates/srTakeover_hook.j2

```diff
@@ -0,0 +1,103 @@
+{% set host = grains['host'] %}
+
+{% for node in hana.nodes %}
+{% if node.host == host and node.primary is defined %}
+
+{% set data = node %}
+{% set dbsid = data.sid.upper() %}
+{% set dbinst = '{:0>2}'.format(data.instance) %}
+{% set dbuser = data.primary.userkey.user_name %}
+{% set dbpwd = data.password %}
+{% set dbversion = salt[hana.get_version](sid=dbsid, inst=dbinst, password=dbpwd) %}
+{% if salt['pkg.version_cmp'](dbversion,'2.0') < 0 %}
+    {% set dbport = ('30'+ dbinst +'15')|int %}
+{% else %}
+    {% set dbport = ('30'+ dbinst +'13')|int %}
+{% endif -%}
+
+#!/usr/bin/env python2
+
+"""
+Auto-generated HA/DR hook script
+
+"""
+
```

Comment:
I would put here directly the parameters like:
```
dbuser="{{ dbuser }}"
dbpwd="{{ dbpwd }}"
...

We don't need to change anything else
```

Reply:
Hi @arbulu89  Sorry I did not understand, do you mean, that we don't need to compute these parameters from pillar file?

Reply:
What @arbulu89 mentioned here is that you don't need to interpolate the values in the entire script. That you can only initialize the variable on the beginning of the script, for example and use the normal python variable in all the occurrences... This way the code is cleaner and easy to change in case of need

Reply:
Exactly, in fact the solution is super straight-forward. The script is ready to only change these 4 lines, nothing else (obviously we need to extract the data you do at the beginning)